### ISSUE TITLE
Set up server rendering + `browserHistory`

### ISSUE THREAD

**Issue by ericsoco** [stamen/bluegreenway#32]
Will this project render on the server? If so, how?

/cc @mojodna

**Comment by emilyhobson** [comment]
Any thoughts around this, @mojodna ?

**Comment by clhenrick** [comment]
setting up server rendering will likely signficantly increase total project time @emilyhobson

**Comment by sethfitz** [comment]
Agreed. I would like to avoid this if at all possible. The main repercussion is SEO.

**Comment by ericsoco** [comment]
@clhenrick we decided to go with `hashHistory`, right? I assume the `phase3` label is essentially marking server rendering as deprioritized. Just catching up here.

**Comment by clhenrick** [comment]
@ericsoco correct, & it seems like it would be easy to go back to `browserHistory` if we end up doing server side rendering.

**Comment by ericsoco** [comment]
Hrm, not sure about "easy". Would require some refactoring. I'd say it's probably a 1-2 day task (not counting the whole serverside rendering thing).
